backend/dal/Helpers/Extensions/EntityEntryExtensions.cs

```diff
@@ -17,23 +17,12 @@ public static class EntityEntryExtensions
         /// <param name="entry"></param>
         public static void UndoAppCreatedEdits(this EntityEntry entry)
         {
-            if (entry.Entity is IDisableBaseAppEntity entity)
+            if (entry.Entity is IBaseAppEntity entity)
             {
-                var oCreatedOn = (DateTime)entry.OriginalValues[nameof(entity.AppCreateTimestamp)];
-                if (!oCreatedOn.Equals(entity.AppCreateTimestamp))
-                    entity.AppCreateTimestamp = oCreatedOn;
-
-                var oCreatedBy = (string)entry.OriginalValues[nameof(entity.AppCreateUserid)];
-                if (String.IsNullOrWhiteSpace(entity.AppCreateUserid) || !oCreatedBy.Equals(entity.AppCreateUserid))
-                    entity.AppCreateUserid = oCreatedBy;
-
-                var oCreatedByKey = (Guid?)entry.OriginalValues[nameof(entity.AppCreateUserGuid)];
-                if (!entity.AppCreateUserGuid.HasValue || !oCreatedByKey.Equals(entity.AppCreateUserGuid))
-                    entity.AppCreateUserGuid = oCreatedByKey;
-
-                var oCreatedByDirectory = (string)entry.OriginalValues[nameof(entity.AppCreateUserDirectory)];
-                if (String.IsNullOrWhiteSpace(entity.AppCreateUserDirectory) || !oCreatedByDirectory.Equals(entity.AppCreateUserDirectory))
-                    entity.AppCreateUserDirectory = oCreatedByDirectory;
+                entry.Property(nameof(IBaseAppEntity.AppCreateTimestamp)).IsModified = false;
+                entry.Property(nameof(IBaseAppEntity.AppCreateUserid)).IsModified = false;
+                entry.Property(nameof(IBaseAppEntity.AppCreateUserGuid)).IsModified = false;
+                entry.Property(nameof(IBaseAppEntity.AppCreateUserDirectory)).IsModified = false;
```

Comment:
Was wondering - didn't @FuriousLlama address this already on his PR?

Reply:
he did, but I needed it as well, and his changes weren't merged yet.

Reply:
Ill deal with the merge conflicts